building/Dockerfile

```diff
@@ -10,7 +10,7 @@
 #     -v $CARGO_TARGET_VOLUME_NAME:/cargo-target:Z \
 #     -v $CARGO_REGISTRY_VOLUME_NAME:/root/.cargo/registry:Z \
 #     -v /path/to/repository_root:/build:Z \
-#     mullvadvpn-app-build ./build.sh
+#     mullvad-app-build ./build.sh
```

Comment:
Why this change? We do use `mullvadvpn-app-build` in all other places.

Reply:
good catch. this is a typo.